Add tip about editor integrations in getting started (#52979)
rails/rails: guides/source/getting_started.md

  things in a web application, and defaults to this set of conventions, rather than
  require that you specify minutiae through endless configuration files.

> Developer environment and editor integrations

Reviewer: For [headings,](https://guides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#headings), we capitalize all words except prepositions, conjunctions, internal articles, and forms of the verb "to be":
```suggestion
Developer Environment and Editor Integrations
```

> ----------------------------

Reviewer 2: ```suggestion
---------------------------------------------
```

> To get the most out of your code editor when working with Rails, you can use a language
> server to provide integrations and rich features such as go to definition, completion,

Reviewer: Maybe also link to a LSP definition?
```suggestion
To get the most out of your code editor when working with Rails, you can use a [language
server](https://en.wikipedia.org/wiki/Language_Server_Protocol) to provide integrations and
rich features such as go to definition, completion,
```

> showing documentation on hover and many others.

Reviewer: We try to use the Oxford Comma.
```suggestion
showing documentation on hover, and many others.
```

> To get the most out of your code editor when working with Rails, you can use a language
> server, which provides integrations and rich features such as go to definition, completion,
> showing documentation on hover and many others.

Reviewer 3: ```suggestion
To get the most out of your code editor when working with Rails, you can use a language
server to provide integrations and rich features such as go to definition, completion,
showing documentation on hover and many others.
```
